runtime/lua/vim/lsp/util.lua

```diff
@@ -683,10 +683,14 @@ end
 
 --- Applies a `WorkspaceEdit`.
 ---
----@param workspace_edit lsp.WorkspaceEdit
+---@param workspace_edit lsp.WorkspaceEdit?
 ---@param offset_encoding 'utf-8'|'utf-16'|'utf-32' (required)
 ---@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workspace_applyEdit
 function M.apply_workspace_edit(workspace_edit, offset_encoding)
+  if not workspace_edit then
+    vim.notify("Language server couldn't provide rename result", vim.log.levels.INFO)
+    return
+  end
```

Comment:
This is not just for rename but also code actions and so on.

Reply:
Ok, I'll just revert this.